api/v1alpha5/metal3datatemplate_types.go

```diff
@@ -175,15 +175,15 @@ type NetworkLinkEthernetMac struct {
 	FromHostInterface *string `json:"fromHostInterface,omitempty"`
 }
 
-// NetworkDataLinkEthernet represents an ethernet link object
+// NetworkDataLinkEthernet represents an ethernet link object.
 type NetworkDataLinkEthernet struct {
 	// +kubebuilder:validation:Enum=bridge;dvs;hw_veb;hyperv;ovs;tap;vhostuser;vif;phy
 	// Type is the type of the ethernet link. It can be one of:
 	// bridge, dvs, hw_veb, hyperv, ovs, tap, vhostuser, vif, phy
 	Type string `json:"type"`
 
 	// Id is the ID of the interface (used for naming)
-	Id string `json:"id"`
+	ID string `json:"id"`
```

Comment:
Bit late on this, but This is an API [change](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api_changes.md) AFAIK for the old API and we should not have touch it..

Reply:
Hmmm true, probably we need to revert it

Reply:
Thats v1a5 btw

Reply:
Ow a bit quick to merge this. I also commented on this a bit up. Didn't squash yet either. But that means at least we can revert only the second commit.